packages/code-studio/src/main/AppMainContainer.tsx

```diff
@@ -1015,10 +995,7 @@ export class AppMainContainer extends Component<
           onChange={this.handleImportLayoutFiles}
           data-testid="input-import-layout"
         />
-        <DebouncedModal
-          isOpen={isDisconnected && !isAuthFailed}
-          debounceMs={1000}
-        >
+        <DebouncedModal isOpen={isDisconnected} debounceMs={1000}>
```

Comment:
I'm curious if this should be pulled out as well. I'd think embed-widget doesn't handle disconnect/reconnect (or doesn't display it at least) since it doesn't handle these events

Reply:
Not sure how this will be shown, right now it does not show up since we have the overlay and the auth failed modal on top of it, so I can't see it regardless.